index.html

```diff
@@ -111,6 +111,60 @@ <h2>A project for first-time contributions</h2>
             </ul>
           </div>
         </div>
+        <!-- ________ *TEMPLATE* Contributor card END ________  -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">NorthOuterTowner</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+              <a href="https://www.twitter.com/your_user_handle" target="_blank">NoTwitter</a>
+              <i class="fab fa-github"></i>
+              <a href="https://www.github.com/NorthOuterTowner" target="_blank">NorthOuterTowner</a>
+          </p>
+          <p class="about">A freshman exploring computer science in Beijing, China.</p>
+                  <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                      <li>
+                        <a href="https://nn.labml.ai/" target="_blank" title="First Resource">Implement AI here!</a>
+                      </li>
+                      <li>
+                        <a href="https://github.com/NorthOuterTowner/C-management" target="_blank" title="Second Resource">My homework last term, could u look it and give me any suggestion?Really Thank u!</a>
```

Comment:
The child element is way too long. Please keep it short.
```suggestion
                        <a href="https://github.com/NorthOuterTowner/C-management" target="_blank" title="Second Resource">My homework last term.</a>
```